packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -709,6 +715,62 @@ export default class PciInstancesAddController {
         );
       }
     }
+    return this.goBack(message, 'success');
+  }
+
+  createNetworkWithGateway() {
+    this.isLoading = true;
+    let openStackId;
```

Comment:
The goal is not to define the variable ?

Reply:
@ghyenne the goal is not to define global variable as I need this variable inside this block only.